### PR TITLE
Add new AR Registries to experimental project

### PR DESCRIPTION
**Pull request by upodroid**
Part of https://github.com/kubernetes/k8s.io/issues/1343

/hold 

We need to create the AR registries in k8s-artifacts-prod and grant IAM roles to k8s-infra-gcr-promoter@k8s-artifacts-prod.iam.gserviceaccount.com service account.

### CODE REVIEW

**Comment by ameukam** [comment]
/approve
leaving lgtm to @kubernetes/release-engineering

**Comment by upodroid** [comment]
thank you, @ameukam cancel the hold once the infra has been deployed

**Comment by upodroid** [comment]
/hold cancel

Looks like the registries are up.

```
 REDACTED  MCW0CDP3YY  ~  ERROR  $  gcrane ls europe-west2-docker.pkg.dev/k8s-artifacts-prod/images
 REDACTED  MCW0CDP3YY  ~  $  gcrane ls europe-west1-docker.pkg.dev/k8s-artifacts-prod/images
 REDACTED  MCW0CDP3YY  ~  $  gcrane ls europe-west1-docker.pkg.dev/k8s-artifacts-prod/image
2022/08/10 20:23:23 No matching credentials were found for "europe-west1-docker.pkg.dev"
Error: GET https://europe-west1-docker.pkg.dev/v2/token?scope=repository%3Ak8s-artifacts-prod%2Fimage%3Apull&service=: DENIED: Permission "artifactregistry.repositories.downloadArtifacts" denied on resource "projects/k8s-artifacts-prod/locations/europe-west1/repositories/image" (or it may not exist)
```

Thanks

File: k8s.gcr.io/manifests/k8s-staging-experimental/promoter-manifest.yaml [hunk 1]
```diff
@@ -8,3 +8,45 @@ registries:
...
+  service-account: k8s-infra-gcr-promoter@k8s-artifacts-prod.iam.gserviceaccount.com
+- name: asia-northeast1-docker.pkg.dev/k8s-artifacts-prod/images/experimental
+  service-account: k8s-infra-gcr-promoter@k8s-artifacts-prod.iam.gserviceaccount.com
+- name: asia-northeast2-docker.pkg.dev/k8s-artifacts-prod/images/experimental
+  service-account: k8s-infra-gcr-promoter@k8s-artifacts-prod.iam.gserviceaccount.com
+- name: australia-southeast1-docker.pkg.dev/k8s-artifacts-prod/images/experimental
+  service-account: k8s-infra-gcr-promoter@k8s-artifacts-prod.iam.gserviceaccount.com
+- name: europe-north1-docker.pkg.dev/k8s-artifacts-prod/images/experimental
+  service-account: k8s-infra-gcr-promoter@k8s-artifacts-prod.iam.gserviceaccount.com
+- name: europe-southeast1-docker.pkg.dev/k8s-artifacts-prod/images/experimental
```
Comment (ameukam) [inline k8s.gcr.io/manifests/k8s-staging-experimental/promoter-manifest.yaml:23]: @upodroid this region don't exist. 

```bash
 gcloud artifacts locations list --project k8s-artifacts-prod | grep 'europe-'
europe-central2
europe-north1
europe-southwest1
europe-west1
europe-west2
europe-west3
europe-west4
europe-west6
europe-west8
europe-west9
```

Comment (upodroid) [reply k8s.gcr.io/manifests/k8s-staging-experimental/promoter-manifest.yaml:23]: thank you

/hold cancel

**Pull request merged**